.gitignore

```diff
@@ -3,3 +3,4 @@
 .pytest_cache
 gtfs/retriever/.idea/
 build
+config.log
```

Comment:
What is this file? I haven't seen it before

Reply:
I don't really know it was created automatcly
That's the file content:
`2019-10-19 09:36:13,358 - default - DEBUG - no_timestamp flag is False
2019-10-19 09:36:13,358 - default - DEBUG - no_md5 flag is False
`

Reply:
Oh, OK. It seems like it from the retriever code, I've never run it before.